examples/api/service/client.go

```diff
@@ -10,20 +11,22 @@ import (
 	peer "github.com/ipfs/go-ipfs/p2p/peer"
 	fsrepo "github.com/ipfs/go-ipfs/repo/fsrepo"
 
-	"golang.org/x/net/context"
+	"code.google.com/p/go.net/context"
 )
 
 func main() {
+
 	if len(os.Args) < 2 {
-		fmt.Println("Please give a peer ID as an argument")
+		fmt.Println("Please give peer ID as an argument")
```

Comment:
i dont think this change is necessary? depending on context, either may be correct.